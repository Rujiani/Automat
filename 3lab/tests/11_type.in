số nguyên  i;
sắp hàng arr = {3};
vuông      cell = { 3 , 4 , thật };

arr[0] = 10;
arr[1] = 20;

kiểm tra loại { arr[0]  i } ;

kiểm tra loại { cell=>busy  arr[1] } ;

kiểm tra loại { cell  arr[0] } ;

kiểm tra loại { kích thước arr  arr[2] } ;
