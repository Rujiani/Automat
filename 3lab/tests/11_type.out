thật
sai
sai
thật
